csrc/host_ir/executor.h

```diff
@@ -11,15 +11,17 @@
 #include <expr_evaluator.h>
 #include <host_ir/container.h>
 #include <host_ir/host_ir.h>
+#ifdef NVFUSER_HOST_IR_JIT
+#include <host_ir/jit.h>
+#endif
+#include <c10/cuda/CUDAStream.h>
```

Comment:
nitpick, let's not put c10 include here. If you add an empty line, linter won't try to mess with it.